### PR TITLE
Improve the disk standby check

### PR DESCRIPTION
**Pull request by miskcoo**
Like #38, this PR improves the standby check in #37. It will check whether the disk is in the standby mode every 1s by default and change the LED color accordingly. 

To enable it, you should do the following things:

```bash
# compile the disk standby checker
g++ -std=c++17 -O2 check-standby.cpp -o ugreen-check-standby

# copy the binary file (the path can be changed, see STANDBY_MON_PATH in ugreen-leds.conf)
cp ugreen-check-standby /usr/bin

# copy the new script and restart the service
cp scripts/ugreen-diskiomon /usr/bin
systemctl restart ugreen-diskiomon
```

Minor changes:
- fix the zpool healthy check when the disk is in the standby mode
- update the debian build scripts
- update the SPEC

Related issue: #21.

### CODE REVIEW

**Comment by miskcoo** [comment]
Hi @gulikoza, can you check whether the new SPEC is correct?

**Comment by gulikoza** [comment]
Hi,

RPM builds successfully and ugreen-check-standby is included.
I suppose the second smartctl check also be removed?

**Comment by miskcoo** [comment]
You are right. I removed it.

File: scripts/ugreen-diskiomon [hunk 1]
```diff
@@ -289,21 +296,7 @@ fi
...
-                if [[ $RET -eq 0 ]]; then
-                    led_color="$COLOR_DISK_HEALTH"
-                    echo $led_color > /sys/class/leds/$led/color
-                elif [[ $RET -gt 1 ]]; then
-                    echo "$COLOR_SMART_FAIL" > /sys/class/leds/$led/color
-                    echo Disk failure detected on /dev/$dev at $(date +%Y-%m-%d' '%H:%M:%S)
-                    continue
-                fi
-            elif [[ "$led_color" != "$COLOR_DISK_HEALTH" ]]; then
+            if ! is_disk_healthy "$led_color"; then
```
Comment (gulikoza) [inline scripts/ugreen-diskiomon:299]: Previous version proceeded with the stat check if the disk was in standby.
Reading `/sys/class/block/${dev}/stat` does not wake the disk, I tested this.

If the disk is in standby and then fails, would the script detect this?

Comment (miskcoo) [reply scripts/ugreen-diskiomon:299]: Yes. I tested in the loop of the zpool healthy check.

`is_disk_healthy` includes both `COLOR_DISK_HEALTH` and `COLOR_DISK_STANDBY`.

Comment (gulikoza) [reply scripts/ugreen-diskiomon:299]: User might not run zpool.
If the disk disappears while in standby, led would not change.

I think that `COLOR_DISK_STANDBY` does not automatically mean `is_disk_healthy`. Sure, we should skip smart check in order not to wake it, but `check disk online status` would need to run for standby disks :)

Comment (miskcoo) [reply scripts/ugreen-diskiomon:299]: I think the code do what you said, but the name `is_disk_healthy` may be misleading -- it should be `is_disk_healthy_or_standby`. 

The subsequent disk online check will run if the color is either `COLOR_DISK_HEALTH` or `COLOR_DISK_STANDBY`. The check is bypassed if the color does not belong to them, which means the disk has been marked as failed somewhere else.

Comment (gulikoza) [reply scripts/ugreen-diskiomon:299]: you are right 👍

**Pull request merged**